packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1569,8 +1597,6 @@ export class Combobox
           onInput={this.inputHandler}
           placeholder={placeholder}
           type="text"
-          // eslint-disable-next-line react/jsx-sort-props -- ref should be last so node attrs/props are in sync (see https://github.com/Esri/calcite-design-system/pull/6530)
-          ref={(el) => (this.textInput = el as HTMLInputElement)}
```

Comment:
I think this is what is breaking tests. 

This needs to remain. See where `textInput` is being used to get the value of the input element. example: `clearInputValue()`